keystorage/yubihsm2/src/test/java/tech/pegasys/signers/yubihsm2/YubiHsm2Test.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.signers.yubihsm2;
+
+import java.util.Optional;
+
+import org.apache.commons.lang3.time.StopWatch;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.Assumptions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+// manual dev test -
+// Make sure yubihsm2 is plugged in
+// for http/https url scheme, make sure connector daemon is running
+// for yhusb://serial=123456 url scheme, no connector daemon is required.
+// uncomment system properties in setup method and modify accordingly
+class YubiHsm2Test {
```

Comment:
This test should probably be marked as disabled so it doesn't get run as part of normal test run